pkgs/development/interpreters/tclreadline/default.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv
+, fetchFromGitHub
+, automake115x
+, autoconf
+, libtool
+, readline
+, tcl
+, tk
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tclreadline";
+  version = "2.3.6";
+
+  src = fetchFromGitHub {
+    owner = "flightaware";
+    repo = "tclreadline";
+    rev = "v${version}";
+    sha256 = "0ik2nr6c9icgmx98zn2vginc89l87xn2d9rmxqjgpqpyfmj14dc3";
+  };
+
+  nativeBuildInputs = [
+    automake115x
+    autoconf
+  ];
+  buildInputs = [
+    readline
+    tcl
+    tk
+  ];
+
+  # Substitute hardcoded /usr/bin/... links with store links
+  postPatch = ''
```

Comment:
@jtojnar does this look fine to you?

Reply:
No. Does not autoconf already do this? What happens if you run `autogen.sh` in `preConfigure`?

Reply:
Yes, with the right arguments it does! Thanks for your help @jtojnar, I'm not really familiar with autoconf.

Force-pushed.